src/status_im/models/browser-history.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im.models.browser-history
+  (:require [re-frame.core :as re-frame]))
+
+(def forget-page-load (atom false))
```

Comment:
why do you use atom but not store it in app-db?

Reply:
Because I wasn't knowledgeable enough about the dispatch system to feel sure the app db will be updated in time after clicking back/forward to forget that page load. If that's not a problem I'll switch it to the app db, the atom is definitely horrible and bad! (I noted this issue in the original to-do) :beers:

Reply:
it should work fine, now it's not idiomatic, when press navigation buttons you should dispatch events and work only with app-db, you can store this `forget-page-load` in browser map, btw why `forget-page-load` ?  maybe `dont-store-history-on-nav-change` or smth?

Reply:
Done! Now stored in `:browser/options :dont-store-history-on-nav-change?` not the atom.